apps/accounts/templates/emails/approve-linked-domain-notify.html

```diff
@@ -0,0 +1,13 @@
+<p>
+Dear {{ provider.name }},
+</p><p>
+Thank you for taking the time to link the domain {{domain}} .
+</p><p>
+We have verified your control of this domain and approved the link: From now on, any searches for {{domain}} in the green web checker will show as green and be attributed to {{provider.name}}.
```

Comment:
We have used _Green Web_ in capitalised form, elsewhere in this message, but here. Woud you use _Green Web Check_ instead? I'll share a link to the controlled vocab spreadsheet that shows some guidance on this.